amazon-sqs/5.9/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc

```diff
@@ -0,0 +1,97 @@
+= Amazon SQS Connector 5.9 Additional Configuration Information - Mule 4
+:page-aliases: connectors::amazon/amazon-sqs-connector-config-topics.adoc
+
```

Comment:
I would add an intro sentence into this doc, something like:

_Set additional configuration for Anypoint Connector for Amazon SQS (Amazon SQS Connector), such as controlling polling behavior globally and locally, receiving messages on the Primary Node Only, creating the FIFO queue, and managing large payloads_